src/classes/DataSchema.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+//******************************************************************************
+//                              DataSchema.php
+//
+// Author(s): Arnaud Charleroy
+// SILEX version 1.0
+// Copyright © - INRA - 2018
+// Creation date: 26 janv. 2018
+// Contact: arnaud.charleroy@inra.fr, anne.tireau@inra.fr, pascal.neveu@inra.fr
+// Last modification date:  26 janv. 2018
+// Subject: A class which represents a dataschema in handsontable
+//******************************************************************************
+
+/**
+ * @link http://www.inra.fr/
+ * @copyright Copyright © INRA - 2018
+ * @license https://www.gnu.org/licenses/agpl-3.0.fr.html AGPL-3.0
+ */
+
+namespace openSILEX\handsontablePHP\classes;
+
+/**
+ * Class which represents DataSchema handsontable option
+ * @author Arnaud Charleroy <arnaud.charleroy@inra.fr>
+ * @since 1.0
+ * @see https://docs.handsontable.com/latest/Options.html#dataSchema
+ */
+class DataSchema implements \JsonSerializable
+{
+
+    /**
+     * @var array contains dataschema value
+     */
+    protected $schema = null;
+
+    public function __construct($schema)
+    {
+        $this->schema = $schema;
+    }
+
+    /**
+     * @example
+     * dataSchema: {id: null, name: {first: null, last: null}, address: null},
+     *
+     * @return string json text
+     */
+    public function jsonSerialize()
```

Comment:
I don't understand why the method is called jsonSerialize() if it returns the attribute $schema. Shouldn't it be called getSchema ?